src/function/scalar/list/list_extract.cpp

```diff
@@ -39,6 +39,14 @@ void ListExtractTemplate(idx_t count, VectorData &list_data, VectorData &offsets
 		if (list_data.validity.RowIsValid(list_index) && offsets_data.validity.RowIsValid(offsets_index)) {
 			auto list_entry = ((list_entry_t *)list_data.data)[list_index];
 			auto offsets_entry = ((int64_t *)offsets_data.data)[offsets_index];
+
+			// 1-based indexing
+			if (offsets_entry == 0) {
```

Comment:
Could we add some tests with indexes/offsets of 0?

Reply:
Done, but maybe I missed some more tricky cases...